PolyEngine/Engine/Src/Engine.hpp

```diff
@@ -164,10 +169,18 @@ namespace Poly
 		/// <returns>Reference to InputQueue instance.</returns>
 		InputQueue& GetInputQueue() { return InputEventsQueue; }
 
+		/// <summary>Returns refference to output queue needed by Main.</summary>
+		/// <returns>Reference to OutputQueue instance.</returns>
+		OutputQueue& GetOutputQueue() { return OutputEventsQueue; }
```

Comment:
Just out of curiosity, what do we output in that queue?

Reply:
Commands to Window handling library (SDL2) in main.cpp such as switching between cursors on panel resizing.